avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -978,7 +1110,79 @@ Describe 'Test file tests' -Tag 'TestTemplate' {
       }
     }
 
-    It "[<moduleFolderName>] Bicep test deployment files should invoke test like [`module testDeployment '../.*main.bicep' = {`]" -TestCases $deploymentTestFileTestCases {
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files should contain a parameter [serviceShort]" -TestCases $deploymentTestFileTestCases {
+
+      param(
+        [object[]] $testFileContent
+      )
+      ($testFileContent -match "^param serviceShort string = '(.*)$") | Should -Not -BeNullOrEmpty -Because 'the module test deployment file should contain a parameter [serviceShort] using the syntax [param serviceShort string = ''*''].'
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files in a [waf-aligned] folder should have a parameter [serviceShort] with a value ending with [min]" -TestCases ($deploymentTestFileTestCases | Where-Object { $_.testFilePath -match '.*[\\|\/]defaults[\\|\/].*' }) {
+
+      param(
+        [object[]] $testFileContent
+      )
+
+      if (($testFileContent | Out-String) -match "param serviceShort string = '(.*)'") {
+        $Matches[1] | Should -BeLike "*min"
+      } else {
+        Set-ItResult -Skipped -Because 'the module test deployment file should contain a parameter [serviceShort] using the syntax [param serviceShort string = ''*min''] but it doesn''t.'
+      }
+    }
+
+    It "[<moduleFolderName>] [<testName>] Bicep test deployment files in a [waf-aligned] folder should have a [serviceShort] parameter with a value ending with  [max]" -TestCases ($deploymentTestFileTestCases | Where-Object { $_.testFilePath -match '.*[\\|\/]max[\\|\/].*' }) {
+
+      param(
+        [object[]] $testFileContent
+      )
+
+      if (($testFileContent | Out-String) -match "param serviceShort string = '(.*)'") {
```

Comment:
As per specs having a max folder is a SHOULD, not a MUST. Should we skip in case no max folder exists?

Reply:
That's actually already the case @eriqua . The test cases are filtered down in the header of the [test](https://github.com/Azure/bicep-registry-modules/pull/606/files#diff-81d3cc7564694721d5e785a1c580040f90ee6189ad3db8eaed4c64942dd9dc49R1134).
You wrote the original test that makes sure we have a `defaults` & `waf-aligned` folder and correctly don't test for `max`.